### DISCUSSION TITLE
How to pass multiple filters using `--autodiscover-filter`?

### DISCUSSION THREAD

**Discussion by artemptushkin** [renovatebot/renovate#23475] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

32.122.0

### If you're self-hosting Renovate, select which platform you are using.

gitlab.com

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

--autodiscover-filter=iptiq/dx/appfw, iptiq/dx/app-template

Expect 2 repositories to be discovered, precisely I need one by regex and another negotiated

### What have you tried so far.

```
#1

--autodiscover-filter="iptiq/dx/appfw iptiq/dx/app-template"

#2

--autodiscover-filter=iptiq/dx/appfw --autodiscover-filter=iptiq/dx/app-template
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by MindTooth** [comment]
Would the following work?

```
--autodiscover-filter="/iptiq/dx/app(fw|-template)/"
```

Ref. https://docs.renovatebot.com/self-hosted-configuration/#autodiscoverfilter

**Reply by artemptushkin** [reply]
yes because it's a single value but why can't I put more than one from the CLI? it's array anyway

**Reply by viceice** [reply]
because the `,` can be part of the regex and it's not easy to split multiple regex

**Reply by artemptushkin** [reply]
maybe it will be easier to accept N parameters for array? `-autodiscover-filter="foo" --autodiscover-filter="baz"`

**Reply by viceice** [reply]
not supported by Commander, our args parser
